### ISSUE TITLE
Uncaught error when .lagoon.yml file is not present results in a very un-landolike experience.

### ISSUE THREAD

**Issue by jonpugh** [lando/lando#2818]
Not sure if this is your bag or not, but I'm posting this in the spirit of smooth developer experience.

## Steps to reproduce

1. Edit the `.lando.yml` file and change the recipe to `recipe: lagoon` in a project with no `.lagoon.yml`
2. Run `lando info`.
3. Receive long confusing error message.

## Expected Behavior

A cool, collected, clear error message like you would expect from a Calrissian.

> Hey, here's what's wrong, friend: You need a ".lagoon.yml" file to use the "lagoon" recipe. Here's a link to the Lagoon/Lando docs to show you the way.

## Error message

ERROR ==> problem loading plugin lando-lagoon from /snapshot/lando/build/cli/integrations/lando-lagoon/app.js: Error: ENOENT: no such file or directory, open '/home/jonpugh/Projects/lagoondemo/.lagoon.yml'
    at Object.openSync (fs.js:440:3)
    at Object.fs.openSync (pkg/prelude/bootstrap.js:491:32)
    at Object.readFileSync (fs.js:342:35)
    at Object.fs.readFileSync (pkg/prelude/bootstrap.js:687:36)
    at Object.loadConfigFiles (/snapshot/lando/build/cli/integrations/lando-lagoon/lib/config.js)
    at /snapshot/lando/build/cli/integrations/lando-lagoon/app.js
    at Plugins.load (/snapshot/lando/build/cli/lib/plugins.js)
    at /snapshot/lando/build/cli/lib/app.js
    at tryCatcher (/snapshot/lando/build/cli/node_modules/bluebird/js/release/util.js:16:23)
    at MappingPromiseArray._promiseFulfilled (/snapshot/lando/build/cli/node_modules/bluebird/js/release/map.js:61:38)
    at MappingPromiseArray.<anonymous> (/snapshot/lando/build/cli/node_modules/bluebird/js/release/promise_array.js:114:31)
    at MappingPromiseArray.init (/snapshot/lando/build/cli/node_modules/bluebird/js/release/promise_array.js:78:10)
    at MappingPromiseArray._asyncInit (/snapshot/lando/build/cli/node_modules/bluebird/js/release/map.js:30:10)
    at _drainQueueStep (/snapshot/lando/build/cli/node_modules/bluebird/js/release/async.js:142:12)
    at _drainQueue (/snapshot/lando/build/cli/node_modules/bluebird/js/release/async.js:131:9)
    at Async._drainQueues (/snapshot/lando/build/cli/node_modules/bluebird/js/release/async.js:147:5) 
[ { service: 'lagooncli',
    urls: [],
    type: 'compose',
    healthy: true,
    config: {},
    version: 'custom',
    meUser: 'www-data',
    hasCerts: false,
    hostnames: [ 'lagooncli.lagoondemo.internal' ] } ]

**Comment by pirog** [comment]
@jonpugh good call

**Comment by jonpugh** [comment]
Very nice!

```sh
$ lando info
ERROR ==> The lagoon recipe requires a lagoon.yml in /home/jon/Projects/demo.lagoon.sh but we couldn't find one! Check out https://docs.lando.dev/config/lagoon.html to see how all the magic works! 
```

Oh, almost... The file has a "dot" in front: `.lagoon.yml`.

**Comment by jonpugh** [comment]
Posted a PR to fix: #2848
